web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts

```diff
@@ -33,18 +33,17 @@ import * as _ from "lodash";
     (stepSelected)="stepSelected($event)"
     (stepUpdate)="updateStep($event)"
     (stepDelete)="deleteStep($event)"
+    (clipboardSuccess)="clipboardSuccess.emit($event)"
   ></app-edit-flow-ui>
 `
 })
 export class EditFlowComponent implements OnInit, OnDestroy {
-
-  @ViewChild(EditFlowUiComponent) editFlowUi: EditFlowUiComponent;
```

Comment:
The ViewChild code was added as part of the following: https://github.com/marklogic/marklogic-data-hub/pull/2689
I don't think it should be removed.